src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -122,7 +122,7 @@ public NotifyFilters NotifyFilter
             set
             {
                 if (((int)value & ~c_notifyFiltersValidMask) != 0)
-                    throw new ArgumentException(SR.Format(SR.InvalidEnumArgument, "value", (int)value, typeof(NotifyFilters).Name));
+                    throw new ArgumentException(SR.Format(SR.InvalidEnumArgument, nameof(NotifyFilter), (int)value, nameof(NotifyFilters)));
```

Comment:
Ok, I got it. I just wanted it to be consistent with the other code.
What about `typeof(NotifyFilters).Name`, should I revert this as well?